Mimble Wimble.md

```diff
@@ -0,0 +1,124 @@
+
+Transactions in MimbleWimble can be combined in a way that makes it hard for outsiders to figure out who sent money to whom. This is similar to mixing several messages so that no one can tell which message came from which sender. This mixing helps make transactions even more private.
+
+However, MimbleWimble can't do some of the tricks that other protocols can, like creating special conditions for transactions or swapping currencies directly without a middleman. But it has a cool trick: as transactions are combined and old, unnecessary data is removed, stopping the history of transactions from growing too big. This keeps things running smoothly without needing a super-powerful computer.
+
+(Animated video explaining Mimblewimble)
+https://www.youtube.com/watch?v=aIh0XjhNgBc
+
+## Elliptic Curve Cryptography (ECC)
+
+Talking a little about the technical part, MimbleWimble uses a form of cryptography known as Elliptic Curve Cryptography (ECC). ECC enables the protocol to confirm the accuracy of transaction amounts and the identities of the sender and receiver, all while keeping this information private. The "discrete" aspect in logarithms, which ECC is based upon, refers to a mathematical field focused on distinct values, incorporating areas like probability and set theory, thereby enhancing the protocol's security.
+
+With the use of the Elliptic Curve Cryptography, Mimblewimble can achieve 3 main goals for its users:
+
+Privacy Protection: MimbleWimble ensures that the transaction details remain a secret between the sender and the receiver. Only they can know how much crypto was sent.
+
+Transaction Legitimacy: It verifies that transactions are valid without the need for those to be signed publicly, maintaining user anonymity.
+
+Enhanced Efficiency:  The protocol achieves greater efficiency and scalability by generating smaller blocks of data. This approach prevents the blockchain from becoming excessively large and difficult to manage over time, a challenge often faced by networks like Bitcoin.
+
+Beyond ECC, MimbleWimble protocol integrates technology used by cryptographic protocols, such as:
+
+Dandelion: This protocol obscures the identities of the sender and receiver, adding another layer of privacy.
+
+CoinJoin: This technique complicates the task of tracking transaction paths. By amalgamating multiple payments from different senders into a single transaction, CoinJoin masks the public addresses involved. This not only enhances privacy but also streamlines blockchain data storage by reducing the need to retain extensive transactional information beyond the essential inputs and outputs.
+
+## Could it work for Decred?
+
+Decred utilizes Stakeshuffle as its privacy protocol. It is responsible for mixing multiple transactions, and hiding who sent coins to whom, without the need for third parties. If you want to learn more about Stakeshuffle click here (https://docs.decred.org/privacy/cspp/overview/?ref=cypherpunktimes.com)!
```

Comment:
"(https://docs.decred.org/privacy/cspp/overview/?ref=cypherpunktimes.com)!" this link looks a little odd, maybe "(https://docs.decred.org/privacy/cspp/overview/)!"